README.md

```diff
@@ -199,50 +242,45 @@ You can activate versioning just by adding a new method to your controller using
 
 So, let's say you have your controllers set up like this:
 
-    site/
-      controllers/
-        __init__.py
+    site/controllers/ __init__.py
 
 and `controllers.__init__.py` contained:
 
-```python
-from endpoints import Controller
+	from endpoints import Controller
 
-class Default(Controller):
-    def GET(self):
-        return "called version 1 /"
-    def GET_v2(self):
-        return "called version 2 /"
+	class Default(Controller):
+   		def GET(self):
+       		return "called version 1 /"
+   		def GET_v2(self):
+       		return "called version 2 /"
 
-class Foo(Controller):
-    def GET(self):
-        return "called version 1 /foo"
-    def GET_v2(self):
-        return "called version 2 /foo"
-```
+	class Foo(Controller):
+    	def GET(self):
+        	return "called version 1 /foo"
+   		def GET_v2(self):
+        	return "called version 2 /foo"
 
 Then, your call requests would be translated like this:
 
-    GET / with Accept: */* -> controllers.Default.GET()
-    GET /foo with Accept: */* -> controllers.Foo.GET()
-
-    GET / with Accept: */*;version=v2 -> controllers.Default.GET_v2()
-    GET /foo with Accept: */*;version=v2 -> controllers.Foo.GET_v2()
-
-Notice how attaching the `;version=v2` to the `Accept` header changes the method that is called to handle the request.
+|    HTTP Request  | Path Followed
+|--------------------------------------------------
+| GET / with Accept: */*			| controllers.Default.GET()
+|GET /foo with Accept: */*			 | controllers.Foo.GET()
+|  GET / with Accept: */*;version=v2		|controllers.Default.GET_v2()
+| GET /foo with Accept: */*;version=v2 	|controllers.Foo.GET_v2()
 
+**Note:** attaching the `;version=v2` to the `Accept` header changes the method that is called to handle the request.
 
 ### CORS support
 
 Endpoints has a `CorsMixin` you can add to your controllers to support [CORS requests](http://www.w3.org/TR/cors/):
 
-```python
-from endpoints import Controller, CorsMixin
+	from endpoints import Controller, CorsMixin
```

Comment:
so none of these code samples needed to be indented 4 spaces because they are fenced with the triple backticks, so they should all be moved back